doc/script_commands.txt

```diff
@@ -2465,9 +2465,9 @@ Examples:
 Returns various information about a player's VIP status.
 
 Valid types:
- 1 - VIP status. (1 if VIP, 0 if non-VIP)
- 2 - VIP expire date. (timestamp string if VIP, 0 if non-VIP)
- 3 - VIP time remaining. (timestamp string if VIP, 0 if non-VIP)
+ VIP_STATUS_ACTIVE - VIP status. (true if VIP, false if non-VIP)
+ VIP_STATUS_EXPIRE - VIP expire date. (timestamp if VIP, 0 if non-VIP)
+ VIP_STATUS_REMAINING - VIP time remaining. (timestamp if VIP, 0 if non-VIP)
```

Comment:
this part
> VIP_STATUS_REMAINING - VIP time remaining. (timestamp if VIP, 0 if non-VIP)

here will be the countdown timer in seconds which shows after how many seconds VIP status will be dropped, not timestamp one.